src/JustSaying/Fluent/ServicesBuilder.cs

```diff
@@ -44,11 +44,6 @@ internal ServicesBuilder(MessagingBusBuilder busBuilder)
         /// </summary>
         internal Func<IMessageLockAsync> MessageLock { get; private set; }
 
-        /// <summary>
-        /// Gets or sets a delegate to a method to create the <see cref="IMessageSerializationFactory"/> to use.
-        /// </summary>
-        internal Func<IMessageSerializationFactory> MessageSerializationFactory { get; private set; }
```

Comment:
I think we'd still want this hook for usage of JustSaying _without_ a DI container.

Reply:
Given that we expose a setter on `SerializationRegister` which takes a dependency on `MessageSerializationFactory ` - I don't think we can even pass this factory in to anywhere that could use it. 

Perhaps we need to change `CreateBus` from:
```csharp
        private JustSayingBus CreateBus(IMessagingConfig config, ILoggerFactory loggerFactory)
        {
            IMessageSerializationRegister register =
                ServicesBuilder?.SerializationRegister?.Invoke() ?? ServiceResolver.ResolveService<IMessageSerializationRegister>();

            return new JustSayingBus(config, register, loggerFactory);
        }
```

to something like this?
```csharp
        private JustSayingBus CreateBus(IMessagingConfig config, ILoggerFactory loggerFactory)
        {
            IMessageSerializationRegister register =
                ServicesBuilder?.SerializationRegister?.Invoke() ??
                ServiceResolver?.ResolveService<IMessageSerializationRegister>() ??
                new MessageSerializationRegister(<subjectProviderFromSomewhere>, 
                    ServicesBuilder.MessageSerializationFactory ?? new NewtonsoftSerializationFactory());

            return new JustSayingBus(config, register, loggerFactory);
        }
```

Do we have tests around using JS without DI? I'm not sure how I'd even go about using it without DI 😁

Reply:
I can't remember if there's an explicit tests for it without one to be honest, but the DI packages build on top of it to plug the dependencies into the service resolver, so it's sort of implicitly tested that way.

Reply:
Ah ok fair. Yeah as for this, it appears that we're able to support a custom `IMessageSerializationFactory` already via the `IMessageSerializationRegister` override - the user could new up a `MessageSerializationRegister` and choose the serialization factory then. I'm not sure how this would work otherwise.